Modules/ZoomInExtension/Resources/script.js

```diff
@@ -1,8 +1,21 @@
 // Wait for the page to load
-document.addEventListener("DOMContentLoaded", function(event) {
+window.addEventListener("load", function(event) {
     // Get the URL from Zoom’s page
-    const url = document.getElementById('scheme_url')['value'];
-
+    let url;
+    let oldZoomPageElement = document.getElementById('scheme_url');
+    if (oldZoomPageElement) {
+        url = oldZoomPageElement['value'];
+    } else {
+        const selectors = document.querySelectorAll('[launch]');
+        if (selectors.length > 0) {
+            url = selectors[0].href;
+        }
```

Comment:
Could we do `querySelector` instead since we only care about one? Also scoped to `a` since we expect an href. Not 100% sure this is the right query syntax though.

```suggestion
        const anchorElement = document.querySelector('a [launch]');
        if (anchorElement) {
            url = anchorElement.href;
        }
```